docs/UserGuide.md

```diff
@@ -66,22 +98,56 @@ Parameter formats:
 - SIZE: One of [`1-room`,`2-room`, `3-room`, `4-room`, `5-room`] (Non case-sensitive).
 - PRICE: `$` followed by a positive integer. e.g. `$150000`
 
-### Viewing help : `help`
+##General Commands
 
-Shows a message explaining how to access the help page.
+### Open Help Window: `help`
+
+Opens a new window that displays information on how to use the app and a URL to the user guide of the app.
 
 Format: `help`
 
+#### Navigating the help window
+
+<img src="images/user-guide/helpWindowUi.png" width="600px">
+
+1. Link to the full user guide.
+2. Buttons to access the different help sections.
+3. Help contents.
+
+### Listing all clients : `list`
+
+Shows a list of all clients in the address book.
+
+Format: `list`
+
+### Clearing all entries : `clear`
+
+Deletes all clients from the address book.
+
+Format: `clear`
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+All clients will be deleted! Use with caution.
+</div>
+
+### Exiting the program : `exit`
+
+Exits the program.
+
+Format: `exit`
+
+##Client Management
+
 ### Adding a client: `add`
 
 Adds a client to RealEstatePro.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [pr/PROPERTY]… [pf/PREFERENCE] [i/FILEPATH:DESCRIPTION]`
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-Either a property or preference must be provided when adding a new user.
+Either a property or preference must be provided when adding a new user and only one of either type can be provided.
```

Comment:
Maybe change `user` to `client` as we reference `user` as the real estate agent right?

Reply:
Instances of `user` where `client` is more apporiate have been changed.